src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-editing.js

```diff
@@ -48,7 +48,7 @@ class IbexaCustomTagEditing extends Plugin {
                     </button>`;
 
                     domElement.innerHTML = `
-                        <div class="ibexa-custom-tag__header-title">${customTagConfig.label}</div>
+                        <div class="ibexa-custom-tag__header-title" title="${customTagConfig.label}">${customTagConfig.label}</div>
```

Comment:
The label should strictly speaking be escaped. If so, how do you want it escaped?
For instance, adding `"` or `</div>` in translation file will break the html

Reply:
Also, changed `title=` into `data-cke-tooltip-text=`. Not sure if this is preferable or not